modules/data.atmosphere/R/temporal.downscaling.R

```diff
@@ -60,7 +60,7 @@ cfmet.downscale.subdaily <- function(subdailymet, output.dt = 1) {
   new.date <- subdailymet[,list(hour = 0:(23 / output.dt) / output.dt),
                     by = c("year", "month", "day", "doy")]
 
-  new.date$date <- new.date[,list(date = lubridate::ymd_h(paste(year, month, day, hour)))]
+  new.date$date <- new.date[,list(date = lubridate::ymd_h(paste(new.date$year, new.date$month, new.date$day, new.date$hour)))]
```

Comment:
I think this line does some data.frame magic that may or may not work with $. Leaving this note as a reminder for me to investigate more.

Reply:
resolved in [32b7afb](https://github.com/PecanProject/pecan/pull/3218/commits/32b7afbb942812a3327acc7658d1089ec1ad56d6)